### PR TITLE
Count cop

### PR DESCRIPTION
**Pull request by rrosenblum**
New performance cop that I thought of the other day when I caught myself making this mistake.

### CODE REVIEW

File: lib/rubocop/cop/performance/count.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop is used to identify usages of `count` on an
+      # `Array` and `Hash` and change them to `size`.
+      #
+      # @example
+      #   # bad
+      #   [1, 2, 3].select { |e| e > 2 }.size
```
Comment (bbatsov) [inline lib/rubocop/cop/performance/count.rb]: You should also account for `length`.

**Comment by rrosenblum** [comment]
I have updated the cop to account for `length`

**Comment by bbatsov** [comment]
You'll also need to rebase this.

**Comment by rrosenblum** [comment]
You are right, I added a number of things to the commit to fix offenses that should have been in the commit for the new cop. I have moved the changes around, updated the commit message, and rebased the code.

**Comment by rrosenblum** [comment]
The code has been updated and rebased.

**Pull request merged**